akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
I'm not sure role-set is more clear than roles, it would also require docs what it means.
I don't have any better suggestion at the moment